### PR TITLE
Peter/docs howto ordinals

### PR DESCRIPTION
**Pull request by peterslany**
Adding docs pages for ordinals and BRC-20 tokens.

### CODE REVIEW

File: docs/docs/build/how-to/ordinals.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Ordinals
+
+Ordinals are a system for tracking and transferring satoshis, Bitcoin's smallest units and attaching data to satoshis. When such attachment of data to satoshi happens an __inscription__ is created. Inscription content is entirely on-chain, stored in taproot script-path spend scripts forever. These scripts receive witness discounts, making inscription content storage relatively economical.
```
Comment (nud3l) [inline docs/docs/build/how-to/ordinals.md:7]: Could you please link to https://docs.ordinals.com/

File: docs/docs/build/how-to/brc-20.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 6
+---
+
+# BRC-20
+
+BRC-20 is a fungible token standard for Bitcoin built on top of ordinals. This standard defines 3 basic functions: `deploy`, `mint` and `transfer`.
```
Comment (nud3l) [inline docs/docs/build/how-to/brc-20.md:7]: Can you please link to https://domo-2.gitbook.io/brc-20-experiment/

File: docs/docs/build/how-to/ordinals.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## How to create an inscription
+Inscriptions are created using taproot script. That means you have to use a P2TR type of address to be able to create the inscription. 
+
+### Wallets that support taproot addresses
+
+- [Sparrow wallet](https://sparrowwallet.com/) (Desktop)
+- [Xverse](https://www.xverse.app/download) (Browser and mobile)
+- [Unisat](https://unisat.io/download ) (Browser)
```
Comment (nud3l) [inline docs/docs/build/how-to/ordinals.md:17]: Would also add the ord wallet https://docs.ordinals.com/guides/inscriptions.html

**Review by nud3l — COMMENTED** [review]
Very nice overall. I left a few minor comments.

**Comment by nud3l** [comment]
@peterslany please resolve the review comments and then this can be merged

**Comment by peterslany** [comment]
@nud3l Comments resolved.

**Review by nud3l — APPROVED** [review]

**Pull request merged**